packages/web-twig/src/Resources/components/Accordion/README.md

```diff
@@ -0,0 +1,198 @@
+      slot="<Pill color='selected'>3</Pill>"
+    >
+      Accordion Header
+    </AccordionHeader>
+    <AccordionContent id="AccordionItemExample1Content" labelId="AccordionItemExample1Header" parent="AccordionExample">
+      Accordion Content
+    </AccordionContent>
+  </AccordionItem>
+</Accordion>
+```
+
+## Usage without lexer:
+
+```twig
+{% embed "@spirit/accordion.twig" with { props: {
+  id: 'AccordionExample',
+} } %}
+    {% block content %}
+        {% embed "@spirit/accordionItem.twig" with { props: {
+          id: `AccordionItemExample`,
+        } } %}
+            {% block content %}
+                {% embed "@spirit/accordionHeader.twig" with { props: {
+                  id: 'AccordionItemExampleHeader',
+                  for: 'AccordionItemExampleContent',
+                  slot: '<Link href="#">Link</Link><Pill color="selected">3</Pill>',
+                  isOpen: true,
+                } } %}
+                    {% block content %}
+                        Accordion Header
+                    {% endblock %}
+                {% endembed %}
+                {% embed "@spirit/accordionContent.twig" with { props: {
+                  id: 'AccordionItemExampleContent',
+                  labelId: 'AccordionItemExampleHeader',
+                  parent: 'AccordionExample',
+                  isOpen: true,
+                } } %}
+                    {% block content %}
+                        Accordion Content
+                    {% endblock %}
+                {% endembed %}
+            {% endblock %}
+        {% endembed %}
+    {% endblock %}
+{% endembed %}
+```
+
+## API
+
+The Accordion itself consists of many components which cannot be used independently.
+
+### Accordion
+
+| Prop name     | Type     | Default     | Required | Description              |
+| ------------- | -------- | ----------- | -------- | ------------------------ |
+| `id`          | `string` | `null`      | no/yes   | Optional, depends on ... |
+| `class`       | `string` | `null`      | no       | Additional class name    |
+| `elementType` | `string` | `'section'` | no       | Custom element type      |
+
+### AccordionItem
+
+| Prop name     | Type     | Default     | Required | Description           |
+| ------------- | -------- | ----------- | -------- | --------------------- |
+| `id`          | `string` | `null`      | yes      | Accordion Item ID     |
+| `class`       | `string` | `null`      | no       | Additional class name |
+| `elementType` | `string` | `'article'` | no       | Custom element type   |
+
+### AccordionHeader
+
+| Prop name     | Type     | Default | Required | Description              |
+| ------------- | -------- | ------- | -------- | ------------------------ |
+| `id`          | `string` | `null`  | yes      | AccordionHeader ID       |
+| `class`       | `string` | `null`  | no       | Additional class name    |
+| `for`         | `string` | `null`  | yes      | Target element selector  |
```

Comment:
We can specifiy the `for` prop is intended for `AccordionContent` ID.

Reply:
You're right, I finished writing it there